docs/source/routing/observability/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -118,6 +118,21 @@ The initial call to Uplink during router startup is not reflected in metrics.
   - `name`: One of `apollo-tracing`, `datadog-tracing`, `jaeger-collector`, `otlp-tracing`, `zipkin-tracing`.
   - `error` = One of `channel closed`, `channel full`.
 
+### Internals
+- `apollo.router.pipelines` - The number of request pipelines active in the router
+  - `schema.id` - The Apollo Studio schema hash associated with the pipeline.
+  - `launch.id` - The Apollo Studio launch id associated with the pipeline (optional).
+  - `config.hash` - The hash of the configuration
+
+### Server
+- `apollo.router.open_connections` - The number of request pipelines active in the router
```

Comment:
```suggestion
- `apollo.router.open_connections` - The number of open connections
```

Reply:
Done, I forced pushed though as I wasn't quite ready

Reply:
np, i wasnt reviewing the code yet, just dropping a driveby comment :)